projects/client/src/app/components/image-field/ImageField.tsx

```diff
@@ -0,0 +1,117 @@
+import ImageOutlinedIcon from '@mui/icons-material/ImageOutlined';
+import React, { useState } from 'react';
+
+type Props = {
+  id: string;
+  children: React.ReactNode;
+  value: string;
+  isDisabled?: boolean;
+  size?: 'small' | 'medium';
+  onFormDataChange: (formData: FormData) => void;
+  isRequired?: boolean;
+  showRequired?: boolean;
+};
+
+function ImageField({ id, children, value, isDisabled, size, onFormDataChange, isRequired, showRequired }: Props) {
+  const [logoCorrect, setLogoCorrect] = useState<boolean>(true);
+  const [previewUrl, setPreviewUrl] = useState<string | null>(null);
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0] || null;
+
+    if (file) {
+      const allowedExtensions = ['png'];
+      const extension = file.name.split('.').pop()?.toLowerCase() || '';
```

Comment:
It is not a big deal, but to have cleaner code, it could be moved to a constant file especifically